### ISSUE TITLE
Whats about the dev branch?

### ISSUE THREAD

**Issue by MacMarde** [xMNG/Microsoft-Rewards-Bot#65]
@ShoGinn What happened to the dev branch? I cant see it anymore.
I did sent you a friendship invitation with discord.

**Comment by revenfar** [comment]
Any resolution to this? I can't find it either

**Comment by andrewjmetzger** [comment]
Hi @MacMarde, @revenfar; thanks for your question! 

TL;DR: Currently there is no `dev` branch, most fixes have been merged into `master`.

[ShoGinn is no longer contributing code to the bot](https://github.com/xMNG/Microsoft-Rewards-Bot/issues/53#issuecomment-507402410).

----

I'm not an expert by any means, so if anyone has more insight, feel free to correct me, but my understanding of the development process is this:

Because of the way git (and by extension, GitHub) works, the latest changes are usually tested and addressed in forked repositories -- [like my own](https://github.com/andrewjmetzger/Microsoft-Rewards-Bot/) -- first, instead of the main one.

In general, once a repository is forked, every issue would get its own branch to keep things organized. Eventually, a pull request ("PR") would be submitted with changes, which would get reviewed by someone in control of the main repository (in this case, xMNG) and (hopefully) merged -- into either the branch `master` for stable releases, or into another branch like `dev` for further testing. 

Once the changes are considered stable, they are usually merged into master, and the branch for the change would be removed (hence, there is no `dev` branch right now).

For an overview of what's currently being worked on, see [Open issues](https://github.com/xMNG/Microsoft-Rewards-Bot/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen), and [Open Pull Requests](https://github.com/xMNG/Microsoft-Rewards-Bot/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen).

To preview the latest changes to the bot, keep an eye on open PRs with the links above and grab code from there. Alternatively, if you have a specific issue that needs solving, let me know and I can tell you which branch to pull instead of `master`.

Anecdotally, xMNG doesn't seem to do testing releases anymore -- all of the code I've contributed was merged directly into master.

(Side note: @xMNG, I would be happy to push to another branch if you want to do pre-release builds or similar; let me know.)

Hope this helps! Please feel free to reply if you have any questions and I'll do my best to answer them. 😃
